docs/medical-api/fhir/overview.mdx

```diff
@@ -122,6 +121,10 @@ Patient using the [Add Patient Data endpoint](/medical-api/api-reference/fhir/ad
 FHIR Resources each have their own specification for properties that may be included in the Resource,
 and they may contain one or more of the [defined Data Types](/medical-api/fhir/data-types).
 
+## Meta
+
+The `meta.source` property in any FHIR resource contains the title of the C-CDA source document from which the resource was derived. You can use this value with the[Get Document URL endpoint](/medical-api/api-reference/document/get-document) to download the original C-CDA document.
```

Comment:
"contains the title of the" -> "points to the"

"value with the" -> space after the `the`